src/test/java/org/sagebionetworks/web/unitclient/widget/profile/UserProfileEditorWidgetImplTest.java

```diff
@@ -225,6 +224,7 @@ public void testConfigure() {
     widget.configure(profile, ORC_ID, mockCallback);
 
     verifyConfigure(1);
+    verify(mockView).setEmails(userEmails, EMAIL2);
```

Comment:
For a normal configure, where user is looking at their own profile, view.setEmails() should be called with both the array of emails and notification email.